ui/components/MesherySettingsPerformanceComponent.js

```diff
@@ -25,6 +25,10 @@ const styles = (theme) => ({
   root: {
     padding: theme.spacing(10),
   },
+  textField: {
+    // minWidth: '190px',
+    // minMargin: 0,
+  },
```

Comment:
@dhruv0000 This is useless I guess? Any reason to have this?

Reply:
Nope. Just kept for any future changes.
Would confuse future contributions :thinking:.
Let me remove it.